vj4/ui/templates/record_detail_status.html

```diff
@@ -0,0 +1,57 @@
+<div class="section visible" id="status">
```

Comment:
I noticed that the original div doesn't contain the "visible" class, and someone adds it automatically. @breeswish